openapi.yaml

```diff
@@ -555,14 +557,19 @@ definitions:
     properties:
```

Comment:
The GET /developers endpoint behaves as if the there is a links object as a property of the `DeveloperResource` object containing a self-link:
```
"data": [
        {
            "type": "developer",
            "id": "21",
            "links": {
                "self": "https://localhost/v1/developers/21"
            },
            "attributes": {
                "name": "Bethesda",
                "website": "https://bethesda.net/en/dashboard"
            }
        }
]
```
However, that links object is not present in the openapi file. The behavior of the endpoint is correct JSON API spec, so the openapi file should be updated.
Here's an [example](https://github.com/osu-mist/directory-api/blob/2f79951ebf62622de88f9f63c699f2cd498d026e/openapi.yaml#L206) of including a self links object in the resource object

Reply:
Added links to DeveloperResource in openapi